css/properties/white-space.json

```
@@ -48,6 +48,54 @@
             "deprecated": false
           }
         },
+        "break-spaces": {
+          "__compat": {
+            "description": "<code>break-spaces</code>",
+            "support": {
+              "chrome": {
+                "version_added": "76"
+              },
+              "chrome_android": {
+                "version_added": "76"
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "69"
+              },
+              "firefox_android": {
+                "version_added": "69"
```

Firefox Android has actually stopped at 68 for now, while they rebuild the application.  Thus, this should be `false`.

```suggestion
                "version_added": false
```